src/step-function.ts

```diff
@@ -370,7 +370,7 @@ export interface $SFN {
 /**
  * @see https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html#asl-intrsc-func-hash-calc
  */
-export type HashAlgorithm = "MD5" | "SHA-1" | "SHA-256" | "SHA-384" | "SHA-512";
+export type HashAlgorithm = ASLGraph.HashAlgorithm;
```

Comment:
Why are there two types?

Reply:
Didn't want to mix the more pubic facing one with the aslgraph one

Reply:
Problem would then be when the user jumps to the definition and has to go look at the private definition. I'd vote for having only the public interface